src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1434,22 +1434,31 @@ protected void AfterMembersChecks(DiagnosticBag diagnostics)
                 compilation.EnsureIsReadOnlyAttributeExists(diagnostics, location, modifyCompilation: true);
             }
 
+            if (hasBaseTypeOrInterface(t => t.ContainsNativeInteger()))
```

Comment:
>if (hasBaseTypeOrInterface(t => t.ContainsNativeInteger())) [](start = 12, length = 59)

Is this fix backed by a unit-test?

Reply:
`AttributeTests_NativeInteger.NoPublicMembers`

---
In reply to: [372701978](https://github.com/dotnet/roslyn/pull/41160#discussion_r372701978) [](ancestors = 372701978)